supervision/classification/core.py

```diff
@@ -40,6 +42,9 @@ def __post_init__(self) -> None:
         _validate_confidence(self.confidence, n)
 
     @classmethod
+    @deprecated(
+        "This method is deprecated and removed in 0.16.0 release. Use sv.Classifications.from_ultralytics() instead."
```

Comment:
can we also quickly explain why `from_yolov8` is being deprecated in favor of `from_ultralytics`?

Reply:
Solved! Albeit these changes belong to https://github.com/roboflow/supervision/pull/278. We will just merge both of these for next release anyway